scripts/docker/run.sh

```diff
@@ -59,30 +65,32 @@ touch $XAUTH
 xauth nlist $DISPLAY | sed -e 's/^..../ffff/' | xauth -f $XAUTH nmerge -
 
 if [ "$os" = "xenial" ]; then
-docker run -it --rm --name astrobee \
-        --volume=$XSOCK:$XSOCK:rw \
-        --volume=$XAUTH:$XAUTH:rw \
-        --env="XAUTHORITY=${XAUTH}" \
-        --env="DISPLAY" \
-        --gpus all \
-      astrobee/astrobee:latest-ubuntu16.04 \
-    /astrobee_init.sh roslaunch astrobee sim.launch $args
+localtag=astrobee/astrobee:latest-ubuntu16.04
+remotetag=ghcr.io/nasa/astrobee:latest-ubuntu16.04
```

Comment:
to simplify might be better to just make localtag=astrobee and remotetag=ghcr.io/nasa, and the rest stays as it was instead of defining these variables 3 times each